src/BugsnagUnity/Native/Android/NativeInterface.cs

```diff
@@ -390,9 +390,13 @@ private Breadcrumb ConvertToBreadcrumb(IntPtr javaBreadcrumb)
             dict.AddToPayload(key, AndroidJNI.CallStringMethod(value, ObjectToString, new jvalue[]{}));
           }
           AndroidJNI.DeleteLocalRef(value);
```

Comment:
Out of interest - why is value deleted here rather than outside the if block?

Reply:
Calling `DeleteLocalRef` on null causes bad times.

Reply:
and none of the other things can be null?

Reply:
Not really no. Given a `Map`, asking for an iterator always returns one. Given an iterator and a `hasNext` check, asking for a `MapEntry` always returns one. Given a `MapEntry`, asking for a `key` always returns one. However, given a `key`, the `value` may be null.